source/release-notes/1.7.txt

```diff
@@ -0,0 +1,30 @@
+.. _c2c-release-notes-1.7:
+
+===============================
+Release Notes for mongosync 1.7
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**Upcoming**
+
+Synchronize Data Between Clusters with Different MongoDB Server Versions
```

Comment:
```suggestion
Synchronize Data Between Clusters with Different MongoDB Server Major Versions
```
I think it would be more accurate to mention different major versions.